lib/material_card/README.md

```diff
@@ -0,0 +1,28 @@
+# Material Card
+
+Material Card is card styling provided by the Google material team also known as
+mdc-web. This is a wrapper to allow the style to be used in AngularDart
+applications. For documentation on how to use these styling please see the [mdc
+documentation][docs] and examples can be found [here][demo].
+
+[docs]: https://github.com/material-components/material-components-web/tree/master/packages/mdc-card
+[demo]: https://material-components-web.appspot.com/card.html
+
+## Setup
+
+The styles are provided by
+`package:angular_components/css/mdc_web/card/mdc-card.scss.css`. To use
+these styles in an angular component simply add it as a `styleUrls` value in
+your `Component` annotation. It is suggested that the style is added before any
+component specific styling so you can easily override style values as needed.
+
+```dart
+@Component(
+    selector: 'my-component',
+    templateUrl: 'my_component.html',
+    styleUrls: const [
```

Comment:
`const` isn't necessary

Reply:
there is now a linter rule that checks redundant `new` and another one for `const`

Reply:
Nice!

Reply:
We aren't ready to remove all the new/const references yet. We can do this one since it is in a README, but we aren't ready to start enforcing it across the package yet.